src/ChainSafe.Gaming.WalletConnect/WalletConnectConfig.cs

```diff
@@ -8,56 +8,122 @@
 
 namespace ChainSafe.Gaming.WalletConnect
 {
+    /// <summary>
+    /// Used for configuring a wallet connect session.
+    /// </summary>
     [Serializable]
-    [JsonObject(MemberSerialization.OptIn)]
+    [JsonObject(MemberSerialization.OptIn)] // Serialize fields explicitly or only ones with [JsonProperty] attribute.
```

Comment:
Can you please move to the `<remarks>` filed of the docstring

Reply:
Yeah that makes more sense, updated.